src/goto-instrument/accelerate/sat_path_enumerator.cpp

```diff
@@ -280,9 +280,9 @@ void sat_path_enumeratort::build_fixed()
       exprt &distinguisher=d->second;
       exprt &shadow=shadow_distinguishers[distinguisher];
 
-      goto_programt::targett assign=fixed.insert_after(fixedt);
-      assign->make_assignment();
-      assign->code=code_assignt(shadow, true_exprt());
+      goto_programt::targett assign = fixed.insert_after(
+        fixedt,
```

Comment:
Unrelated to this PR: we should avoid variable names that look like types (according to our naming convention). Several occurrences.